data/json/recipes/electronics/components.json

```diff
@@ -0,0 +1,65 @@
+[
+  {
+    "type" : "recipe",
+    "result": "amplifier",
+    "category": "CC_ELECTRONIC",
+    "subcategory": "CSC_ELECTRONIC_COMPONENTS",
+    "skill_used": "electronics",
+    "difficulty": 1,
+    "time": 30000,
+    "book_learn": [ [ "radio_book", 3 ] , [ "manual_electronics", 1 ] ],
+    "decomp_learn": 1,
+    "qualities" : [
+      { "id": "SCREW", "level": 1 },
+      { "id": "MULTIMETER", "level": 2 }
```

Comment:
I'm skeptical that a multimeter is strictly required for assembling an amplifier, it's in "faster if you have one" territory, but otherwise you should be able to do something janky like find a led and resistor that has the appropriate resistance for voltage, something similar for current, etc.